src/main/scala/com/typesafe/conductr/sbt/TypesafeConductR.scala

```diff
@@ -30,47 +31,58 @@ private[conductr] object TypesafeConductR {
   val conductrAttrKey = AttributeKey[ActorRef]("sbt-typesafe-set-conductr-task")
   val actorSystemAttrKey = AttributeKey[ActorSystem]("sbt-typesafe-setConductrTask-actor-system")
 
+  def setControlServer(host: sbt.URL, s: State, log: Logger): String =
+    withConductRController(s) { conductr =>
+      log.info(s"Setting Control Server URL to $host")
+      val request = SetControlServer(host.toURI)
+      Try(
+        Await.result(conductr.ask(request)(2 seconds).mapTo[String], 2 seconds)
```

Comment:
Why not use `Await.ready` and return `Unit` instead of the obviously not used string?